composer.json

```diff
@@ -30,7 +30,6 @@
         "php-http/guzzle6-adapter": "^1.1",
```

Comment:
Not sure, who requires these libraries php-http/guzzle6-adapter and php-http/httplug-bundle. Doi Util uses guzzlehttp/guzzle bundle for making HTTP API requests

Reply:
looks like just the httphotplugbundle which is removed below. :+1:

Reply:
The plug bundle is still there. I believe plug is either recommended or required by guzzle.